### PR TITLE
fix signIn page

### PR DESCRIPTION
**Pull request by SergeyShynkarenko**

### CODE REVIEW

File: src/pages/SignIn/components/SignInForm/index.js [hunk 1]
```diff
@@ -1,73 +1,53 @@
...
-          startAdornment: (
-            <InputAdornment position="start">
-              <MailOutlineIcon />
-            </InputAdornment>
-          ),
-        }}
-      />
+        <TextField
+          placeholder='Email'
+          sx={{ my: '15px' }}
```
Comment (IShinkarenko) [inline src/pages/SignIn/components/SignInForm/index.js]: No!
Remove `sx={{ my: '15px' }}`

What if you will have 10 fields?

you have to use Stack component from MUI - https://mui.com/material-ui/react-stack/#interactive

Screenshot 2023-01-03 at 22 39 32

File: .eslintrc.json [hunk 2]
```diff
@@ -3,22 +3,14 @@
     "browser": true,
     "es2021": true
   },
-  "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react-hooks/recommended",
```
Comment (IShinkarenko) [inline .eslintrc.json:10]: we need this one `eslint-plugin-react-hooks`
don't forget to install

File: .eslintrc.json [hunk 3]
```diff
@@ -3,22 +3,14 @@
...
-    "plugin:import/recommended",
```
Comment (IShinkarenko) [inline .eslintrc.json:11]: and this `plugin:import/recommended`
don't forget to install

**Review by IShinkarenko — APPROVED** [review]

File: src/pages/SignIn/components/SignInForm/index.js (hunk 1 above)
Comment (SergeyShynkarenko) [reply src/pages/SignIn/components/SignInForm/index.js]: Fixed

File: .eslintrc.json (hunk 2 above)
Comment (SergeyShynkarenko) [reply .eslintrc.json:10]: Added

File: .eslintrc.json (hunk 3 above)
Comment (SergeyShynkarenko) [reply .eslintrc.json:11]: Added

**Review by IShinkarenko — APPROVED** [review]

**Pull request merged**